src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -27,6 +27,10 @@ public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
 
+    public static String innerTrim(String string) {
+        return String.join(" ", string.split("\\s+"));
```

Comment:
Okay so \s matches more than just spaces, like newlines. But ig we can just say we arent supporting any of the other whitespaces for now so im going to approve this trim.

However, can you edit it to `String.join(" ", string.split("\\s+")).trim();`? This is to handle when the string is like "  uwu"